RNReanimated.podspec

```diff
@@ -97,8 +62,53 @@ Pod::Spec.new do |s|
     s.dependency 'hermes-engine'
   end
   s.dependency 'React-callinvoker'
-
-  if config[:react_native_minor_version] >= 72 && !fabric_enabled
+  if config[:react_native_minor_version] >= 72 && !new_arch_enabled
     s.dependency 'React-RCTAppDelegate'
   end
 end
+end
+
+Pod::Spec.new do |s|
+  
+  s.name         = "RNReanimated"
+  s.version      = reanimated_package_json["version"]
+  s.summary      = reanimated_package_json["description"]
+  s.description  = <<-DESC
+                  RNReanimated
+                   DESC
+  s.homepage     = "https://github.com/software-mansion/react-native-reanimated"
+  s.license      = "MIT"
+  # s.license    = { :type => "MIT", :file => "FILE_LICENSE" }
+  s.author       = { "author" => "author@domain.cn" }
+  s.platforms    = { :ios => ios_min_version, :tvos => "9.0", :osx => "10.14" }
+  s.source       = { :git => "https://github.com/software-mansion/react-native-reanimated.git", :tag => "#{s.version}" }
+
+  s.source_files = [
+    "apple/**/*.{mm,h,m}",
+    "Common/cpp/**/*.{cpp,h}"
+  ]
+
+  s.preserve_paths = [
+    "Common/cpp/hidden_headers/**"
+  ]
+
+  s.pod_target_xcconfig = {
+    "USE_HEADERMAP" => "YES",
+    "DEFINES_MODULE" => "YES",
+    "HEADER_SEARCH_PATHS" => "\"$(PODS_TARGET_SRCROOT)/ReactCommon\" \"$(PODS_TARGET_SRCROOT)\" \"$(PODS_ROOT)/RCT-Folly\" \"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/DoubleConversion\" \"$(PODS_ROOT)/Headers/Private/React-Core\" \"$(PODS_ROOT)/Headers/Private/Yoga\"",
+    "FRAMEWORK_SEARCH_PATHS" => "\"${PODS_CONFIGURATION_BUILD_DIR}/React-hermes\"",
+    "CLANG_CXX_LANGUAGE_STANDARD" => "c++17",
+  }
+  s.compiler_flags = folly_compiler_flags + ' ' + boost_compiler_flags + ' -DHERMES_ENABLE_DEBUGGER'
+  s.xcconfig = {
+    "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\" \"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/glog\" \"$(PODS_ROOT)/RCT-Folly\" \"$(PODS_ROOT)/Headers/Public/React-hermes\" \"$(PODS_ROOT)/Headers/Public/hermes-engine\" \"$(PODS_ROOT)/#{config[:react_native_common_dir]}\"",
+    "OTHER_CFLAGS" => "$(inherited)" + " " + folly_flags + " " + fabric_flags + " " + example_flag + " " + version_flag + " " + debug_flag
+  }
+  s.requires_arc = true
+  s.dependency "ReactCommon/turbomodule/core"
```

Comment:
do we need this dependency in both cases?

Reply:
Yes, we need it :/

Reply:
😢